Xamarin.Forms.Platform.iOS/Renderers/EntryRenderer.cs

```diff
@@ -239,7 +239,7 @@ void UpdateAlignment()
 			Control.TextAlignment = Element.HorizontalTextAlignment.ToNativeTextAlignment(((IVisualElementController)Element).EffectiveFlowDirection);
 		}
 
-		protected internal virtual void UpdateColor()
+		private protected virtual void UpdateColor()
```

Comment:
Just need to sanity check that this member was not exposed publicly in a previous release.